src/Fixer/Comment/NoEmptyCommentFixer.php

```diff
@@ -58,34 +67,105 @@ public function getPriority()
     }
 
     /**
+     * {@inheritdoc}
+     */
+    public function isCandidate(Tokens $tokens)
```

Comment:
but the methods same visibility are now in alphabetic order, you want `isCandidate` always above `fix` method ?